src/openrct2-ui/title/TitleSequencePlayer.cpp

```diff
@@ -317,12 +317,12 @@ class TitleSequencePlayer final : public ITitleSequencePlayer
         return true;
     }
 
-    void SetViewZoom(const uint32_t& zoom)
+    void SetViewZoom(int8_t zoom)
```

Comment:
Why not typehint ZoomLevel here?

Reply:
I was a bit hesitant with this one since it uses the value from `TitleCommand` directly, and I'm not sure how that gets loaded exactly. Looks like there is just one call for it, so just updating this function should be easy, though I'll leave `TitleCommand::Zoom` untouched for now.